test/src/integration/chain.test.ts

```diff
@@ -399,9 +399,6 @@ describe("chain", function() {
         });
 
         const data = tx.toJSON();
-        data.data.output.lockScriptHash = `0x${
-            data.data.output.lockScriptHash
-        }`;
```

Comment:
Was this a workaround for the bug of `Transaction.toJSON()`?

Reply:
Yes. I think so.